arenadata/Dockerfile.ubuntu

```
@@ -39,7 +39,8 @@ RUN mkdir /home/gpadmin/bin_gpdb
 
 ENV TARGET_OS=ubuntu
 ENV OUTPUT_ARTIFACT_DIR=bin_gpdb
-ENV CONFIGURE_FLAGS="--enable-debug-extensions --with-gssapi --enable-cassert --enable-debug --enable-depend" 
+# When there are both of python2 and python3 we should set UNICODE size for python2
+ENV CONFIGURE_FLAGS="--enable-debug-extensions --with-gssapi --enable-cassert --enable-debug --enable-depend CFLAGS=-DPy_UNICODE_SIZE=4"
```

Maybe this will be a little more readable?
```suggestion
ENV CFLAGS="-DPy_UNICODE_SIZE=4"
ENV CONFIGURE_FLAGS="--enable-debug-extensions --with-gssapi --enable-cassert --enable-debug --enable-depend $CFLAGS"
```

Also, can't we change `gpdb/gpMgmt/bin/Makefile` to build `subprocess32` with that define instead of modifying the image, without breaking anything else?

---

we can, but it does not need for all cases.